src/gui/addnewtorrentdialog.cpp

```diff
@@ -382,23 +385,20 @@ void AddNewTorrentDialog::updateDiskSpaceLabel()
     QString size_string = torrent_size ? Utils::Misc::friendlyUnit(torrent_size) : QString(tr("Not Available", "This size is unavailable."));
     size_string += " (";
     size_string += tr("Free space on disk: %1").arg(Utils::Misc::friendlyUnit(Utils::Fs::freeDiskSpaceOnPath(
-                                                                   ui->savePathComboBox->itemData(
-                                                                       ui->savePathComboBox->currentIndex()).toString())));
+                                                                   ui->savePath->selectedPath())));
     size_string += ")";
     ui->size_lbl->setText(size_string);
 }
 
-void AddNewTorrentDialog::onSavePathChanged(int index)
+void AddNewTorrentDialog::onSavePathChanged(const QString &/*text*/)
```

Comment:
I mean you can turn it to `void AddNewTorrentDialog::onSavePathChanged()` unless you need `QString` param.